<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1648</width>
    <height>798</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1648</width>
    <height>798</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>楷体</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>10</y>
     <width>400</width>
     <height>441</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>楷体</family>
    </font>
   </property>
   <property name="title">
    <string>操作区</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QPushButton" name="openfilebutton">
      <property name="text">
       <string>打开文件</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>文件路径：</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="filepathlineEdit"/>
    </item>
    <item>
     <widget class="QPushButton" name="gainpushButton">
      <property name="text">
       <string>计算信息增益</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="savegainpushButton">
      <property name="text">
       <string>保存信息增益</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="treemakepushButton">
      <property name="text">
       <string>生成决策树</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>输入西瓜个数：</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="xiguanumlineEdit"/>
    </item>
    <item>
     <widget class="QPushButton" name="randompushButton">
      <property name="text">
       <string>生成随机数据</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="judgepushButton">
      <property name="text">
       <string>判定</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>耗时：</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="timelineEdit"/>
    </item>
    <item>
     <widget class="QLabel" name="label_6">
      <property name="text">
       <string>                                              s</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="savedatapushButton">
      <property name="text">
       <string>保存</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="verticalSpacer_2">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
   <zorder>label_3</zorder>
   <zorder>filepathlineEdit</zorder>
   <zorder>openfilebutton</zorder>
   <zorder>treemakepushButton</zorder>
   <zorder>randompushButton</zorder>
   <zorder>judgepushButton</zorder>
   <zorder>timelineEdit</zorder>
   <zorder>savedatapushButton</zorder>
   <zorder>gainpushButton</zorder>
   <zorder>label_6</zorder>
   <zorder>xiguanumlineEdit</zorder>
   <zorder>label_4</zorder>
   <zorder>label_5</zorder>
   <zorder>verticalSpacer_2</zorder>
   <zorder>savegainpushButton</zorder>
  </widget>
  <widget class="QGroupBox" name="datagroupBox_2">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>10</y>
     <width>1221</width>
     <height>781</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>700</width>
     <height>0</height>
    </size>
   </property>
   <property name="title">
    <string>数据区</string>
   </property>
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="2" column="0">
     <widget class="QTableWidget" name="pridatatableWidget">
      <property name="minimumSize">
       <size>
        <width>700</width>
        <height>0</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QTableWidget" name="randomdatatableWidget">
      <property name="minimumSize">
       <size>
        <width>700</width>
        <height>0</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>随机数据</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>原始数据</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QTableWidget" name="gaintableWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>480</y>
     <width>301</width>
     <height>301</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>460</y>
     <width>72</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>信息增益</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
